GoogleForCreators/web-stories-wp
# Visual revamp: second pass (pills, slider, more)

## Summary



Addresses [comments from the first pass](https://github.com/google/web-stories-wp/pull/3622#issuecomment-669222773) and continues implementing the new design.

## Relevant Technical Choices

N/A

## To-do

In future PRs: Icons, masonry design, moving the toolbar, background media changes.

## User-facing changes

#### Corrected colors
Corrected colors in multiple places to reflect [Sam's comments](https://github.com/google/web-stories-wp/pull/3622#issuecomment-669222773)

<img width="282" alt="Screen Shot 2020-08-10 at 11 15 05 PM" src="https://user-images.githubusercontent.com/591655/89873957-0b434700-db70-11ea-986b-1259da500fe1.png">

#### Switch + Slider
<img width="270" alt="Screen Shot 2020-08-10 at 11 15 19 PM" src="https://user-images.githubusercontent.com/591655/89874089-388ff500-db70-11ea-8529-6bf9fb5b275d.png">

#### Pills + Tabs + Search
<img width="361" alt="Screen Shot 2020-08-11 at 12 20 11 AM" src="https://user-images.githubusercontent.com/591655/89874124-4776a780-db70-11ea-86b9-7b301072dde8.png">

#### Outlined shapes
<img width="353" alt="Screen Shot 2020-08-11 at 12 53 53 AM" src="https://user-images.githubusercontent.com/591655/89874197-5f4e2b80-db70-11ea-84dc-ccdf18bd3492.png">


#### Text presets: more options
<img width="348" alt="Screen Shot 2020-08-11 at 12 30 51 AM" src="https://user-images.githubusercontent.com/591655/89874030-21510780-db70-11ea-9979-522510c3741b.png">


## Testing Instructions

Visual changes should be present on the editor as well as on Storybook

---



Partial for #3641

See #4058
See #4061
See #4060
